validation/Validation.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+#include <Global.test.hpp>
+
+class ValidationTestDefinitions : public ::testing::Test
+{
+   protected:
+    struct TestCase
+    {
+        std::string testName_;
+        std::vector<std::string> compareWith_;
+        std::vector<double> tolerance_;
```

Comment:
break down into explicit names for tolerances for now, otherwise kinda hard to know what tol we are talking about